api/src/main/java/org/openmrs/module/biometric/api/contract/IdentifierResponse.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+ * the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * OpenMRS is also distributed under the terms of the Healthcare Disclaimer located at
+ * http://openmrs.org/license.
+ *
+ * <p>Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS graphic logo is
+ * a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.biometric.api.contract;
+
+public class IdentifierResponse {
+
+  private String participantUuid;
+
+  private String identifierTypeName;
+
+  private String identifierValue;
+
+  public IdentifierResponse(
+      String participantUuid, String identifierTypeName, String identifierValue) {
```

Comment:
Thanks @druchniewicz  , There is minor indentation to fix. keeping everything on one single line

Reply:
this is automatic formatting made by plugin that we use in CfL (google-java-code formatter)